packages/components/stories/Notification.js

```diff
@@ -27,32 +26,38 @@ class NotificationWrapper extends React.Component {
 					},
 				},
 			]);
-			this.setState({	counter: this.state.counter + 1 });
+			this.setState(prevState => ({ counter: prevState.counter + 1 }));
```

Comment:
to not define the callback function here because you don't need any scoped variable